src/containerapp/azext_containerapp/tests/latest/test_containerapp_up.py

```diff
@@ -0,0 +1,29 @@
+import os
+from unittest import mock
+import time
+import requests
+
+from azure.cli.testsdk.reverse_dependency import get_dummy_cli
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
+from knack.util import CLIError
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@live_only()
+class ContainerAppUpImageTest(ScenarioTest):
+    @ResourceGroupPreparer(location="eastus2")
+    def test_containerapp_up_image_e2e(self, resource_group):
+        env_name = self.create_random_name(prefix='env', length=24)
+        self.cmd(f'containerapp env create -g {resource_group} -n {env_name}')
+        image = "mcr.microsoft.com/azuredocs/containerapps-helloworld:latest"
```

Comment:
@haroonf in the next release lets update this test to not include the tag - since we had a bug on this a while back would be good to make that the test case. Will file a bug on you for this - not a blocker. Thanks!